src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,31 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          <span> Posted by </span>
+
+          <UserInfo user={user} />
```

Comment:
check if the user exists

```suggestion
          {user && <UserInfo user={user} />}
```